lib/wovnrb/api_translator.rb

```diff
@@ -111,7 +130,7 @@ def compress_request_data(data_hash)
     end
 
     def api_uri
-      Addressable::URI.parse("#{@store.settings['api_url']}/v0/")
+      Addressable::URI.parse("#{@store.settings['api_url']}/v0")
```

Comment:
The `api_url` setting itself wasn't checked for trailing slash but I guess that's not too important because customers  probably would never set an alternative setting anyway.